cli/src/main/java/org/jboss/as/cli/impl/CliLauncher.java

```diff
@@ -165,6 +165,8 @@ public static void main(String[] args) throws Exception {
                     ctxBuilder.setEchoCommand(true);
                 } else if (arg.equals("--output-json")) {
                     ctxBuilder.setOutputJSON(true);
+                } else if (arg.equals("--color-output")) {
```

Comment:
I don't think you updated help for this new option. That is located in https://github.com/jfdenise/wildfly-core/blob/security-auth/cli/src/main/resources/help/help.txt